src/main/java/com/minecolonies/coremod/items/ItemBreadDough.java

```diff
@@ -0,0 +1,21 @@
+package com.minecolonies.coremod.items;
+
+import com.minecolonies.api.creativetab.ModCreativeTabs;
+
+import static com.minecolonies.api.util.constant.Constants.STACKSIZE;
+
+/**
+ * Class handling the Santa hat.
```

Comment:
copy paste error/ also for the other items

Reply:
Yup. Will clean up.